Emby.Drawing.Skia/StripCollageBuilder.cs

```diff
@@ -99,56 +99,48 @@ private SKBitmap BuildThumbCollageBitmap(string[] paths, int width, int height)
                         using (var resizeBitmap = new SKBitmap(iWidth, iHeight, currentBitmap.ColorType, currentBitmap.AlphaType))
                         {
                             currentBitmap.Resize(resizeBitmap, SKBitmapResizeMethod.Lanczos3);
-                            // determine how much to crop
+                            // crop image
```

Comment:
In my opinion this should be split into a private method instead of using a comment to document what's happening